.github/workflows/pre-commit-autoupdate.yml

```
@@ -0,0 +1,63 @@
+# Copyright (c) 2023 Sebastian Pipping <sebastian@pipping.org>
+# Licensed under the Apache License Version 2.0
+
+name: Keep pre-commit hooks up to date
+
+on:
+  schedule:
+    - cron: '0 16 * * 5'  # Every Friday 4pm
+  workflow_dispatch:
```

Nit-picky, but I don't care for empty assignments, change to an empty dict.
```suggestion
  workflow_dispatch: {}
```

---

I see little value in that personally:
- it works well as is and is not "wrong" in any semantic sense
- and I see no use of dicts in official docs but instead I do see empty assignments, e.g. at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-of-jobsjob_idsecretsinherit
- it would be inconsistent with what the two existing workflows have

So I consider this addition of noise, personally.